src/content/reference/rules/index.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Rules of React
+---
+
+<Intro>
+Just as different programming languages have their own ways of expressing concepts, React has its own idioms — or rules — for how to express patterns in a way that is easy to understand and yields high-quality applications. This page lists all the Rules of React.
```

Comment:
> This page lists all the Rules of React.

I think this should be removed. It would be easy for someone to misinterpret this as saying that the rules of React won't ever change/be remove/added/evolve or if you follow these rules your code will always be good.